health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreenParent/DrawerFieldComposer.js

```
@@ -0,0 +1,288 @@
+    case "min":
+    case "max":
+    case "numberLength":
+      return "number";
+    case "startDate":
+    case "endDate":
+      return "date";
+    case "countryPrefix":
+      return "prefix";
+    default:
+      return false;
+      break;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)